library/src/scala/util/break.scala

```diff
@@ -0,0 +1,20 @@
+package scala.util
+
+/** This object has two apply methods that abort the current computation
+ *  up to an enclosing `boundary` call.
+ */
+object break:
+
+  /** Abort current computation and instead return `value` as the value of
+   *  the enclosing `boundary` call that created `label`.
+   */
+  transparent inline def apply[T](value: T)(using l: boundary.Label[T]): Nothing =
```

Comment:
In fact, if it was not even `inline` at all, and both `Break`'s constructor and `Label.break` were `private[util]`, the `dropBreaks` phase would not have to deal with `Inlined` nodes + `Throw`s with various indirections. It could directly match on `break.apply(...)` instead.

Reply:
Yes, they are now normal methods, too.